README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Consider including steps for verifying that `go-task` has been installed successfully after running the installation command, which can help users confirm that the setup was successful.